### ISSUE TITLE
Project budget document

### ISSUE THREAD

**Issue by publishwhatyoufund** [pwyf/2024-Index-indicator-definitions#11]
## Description

This is a specific budget detailing what the intended spending is for the different lines of the individual activity.

This can be either at the activity or transaction level, and is only expected if the activity is in the implementation, completion or post-completion phase.

This is not expected if the aid type is budget support (A01 or A02) or if the default aid type code is administrative costs (G01).

## Current test

```python
Budget document is present
    Given an IATI activity
     And the activity is current
     And `activity-status/@code` is one of 2, 3 or 4
     And `default-aid-type/@code` is not any of A01, A02 or G01
     And `transaction/aid-type/@code` is not any of A01 or A02
     Then `document-link/category[@code="A05"]` should be present
```

**Comment by YohannaLoucheur** [comment]
This test should exclude all core funding (B01, B02) and pooled funding, trust funds etc (B03)

**Comment by publishwhatyoufund** [comment]
The project budget document is looking for a specific budget detailing what the intended spending is for the different lines of the individual activity.

It is important to note that this test is not prescriptive about the level of detail required in the project break down. It can be across years or sectors or any other detail.

In addition, the budget can be part of an existing document which outlines a break down of funds. For core funds this could be, for example, the terms of reference detailing how core funds will be administered to the on-funding agency. 

As in previous consultations when exclusions have been requested for the project budget document we have found examples when we have surveyed publishers’ data that some donors are including legitimate budget breakdowns with their core contributions (B aid types).
Since it is possible and more transparent to publish budgets for these activities we will not exclude these aid types.
